### PR TITLE
Bugfix: slave ports not coming up when LAG is created through opx-con…

### PR DESCRIPTION
**Pull request by rakeshdatta**
Bugfix: slave ports not coming up when LAG is created through opx-config-lag

Signed-off-by: Rakesh Datta <rakesh.datta@dell.com>

### CODE REVIEW

**Comment by rakeshdatta** [comment]
Tagging the respective github issue: #8

File: bin/opx-config-lag [hunk 1]
```diff
@@ -116,14 +109,6 @@ class cli(object):
...
 
 
-        # Assign unblocked ports
-        if len(self.unblocked_port_list) != 0:
-            self.cps_cb('set', 
-                        {attr_intf_name:self.args.name, 
-                         attr_lag_unblocked_ports:sorted(self.unblocked_port_list)
-                        }
-                        )
-
```
Comment (atanu-mandal) [inline bin/opx-config-lag:126]: Is the default mode of member port is unblocked ? Please clarify.

File: bin/opx-config-lag [hunk 2]
```diff
@@ -116,14 +109,6 @@ class cli(object):
         self.set_member_ports(sorted(member_ports),'set')
 
 
-        # Assign unblocked ports
```
Comment (jeffyin-dell) [inline bin/opx-config-lag:119]: What happened to the unblocked ports? If you're getting rid of this section, then shouldn't you also remove it from the list of args?

Comment (rakeshdatta) [reply bin/opx-config-lag:119]: @jeff-yin 
- For the create operation, any member port is by-default unblocked. So we need not explicitly add them to the unblocked-port list. 
- unblockedports argument is still needed. Without that we cant specify the ports which are 'not' blocked.

**Review by jeffyin-dell — APPROVED** [review]

**Review by atanu-mandal — APPROVED** [review]

File: bin/opx-config-lag (hunk 1 above)
Comment (rakeshdatta) [reply bin/opx-config-lag:126]: Yes.

**Pull request merged**